src/index.scss

```diff
@@ -0,0 +1,231 @@
+      list-style: none;
+      display: flex;
+      padding: 0;
+      margin: 0;
+    }
+
+    &__item {
+      position: relative;
+      margin-left: $spacing-medium;
+    }
+
+    &__link {
+      text-decoration: none;
+      color: $text-color-dark;
+      text-transform: uppercase;
+      font-weight: 500;
+      padding: 10px 0;
+
+      &::after {
+        content: '';
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: -5px;
+        height: 2px;
+        background-color: $primary-color;
+        display: none;
+      }
+
+      &:hover,
+      &.is-active {
+        transition: $hover-transition;
+        color: $primary-color;
+      }
+
+      &:hover::after,
+      .nav__item:first-child &::after {
+        transition: $hover-transition;
+        display: block;
+      }
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  height: $card-height;
+  border: 1px solid $secondary-color;
+  border-radius: $border-radius;
+  transition: $hover-transition;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    background-image: url(./images/imac.jpeg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 160px;
+    height: 134px;
+    margin: $spacing-large $spacing-medium;
+  }
+
+  &__title {
+    font-family: $font-family-roboto;
+    font-size: $font-size-medium;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $text-color-dark;
+    width: 166px;
+    height: 36px;
+    margin-left: $spacing-medium;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-family: $font-family-roboto;
+    font-size: $font-size-small;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $text-color-light;
+    margin-left: $spacing-medium;
+    margin-bottom: 15px;
+  }
+
+  &__reviews-wrapper {
+    display: flex;
+    width: 166px;
+    margin-left: $spacing-medium;
+    margin-bottom: 20px;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-family: $font-family-roboto;
+    font-size: $font-size-small;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  &__price-info {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: $spacing-medium;
+    margin-bottom: 20px;
+  }
+
+  &__text {
+    font-family: $font-family-roboto;
+    font-size: $font-size-medium;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__price {
+    font-family: $font-family-roboto;
+    font-size: $font-size-large;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    margin-bottom: $spacing-medium;
+    margin-left: $spacing-medium;
+    background-color: $primary-color;
+    border-radius: $border-radius;
```

Comment:
Your button have a `border` that is inherit from the tag `button`, you should deal with it, either changing the color of the border to the color you need or using `border: none;` to remove it.